climpred/tests/test_perfectModelEnsemble_class.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from climpred import PerfectModelEnsemble
+from climpred.tutorial import load_dataset
+
+
+@pytest.fixture
+def PM_da_ds1d():
```

Comment:
I know the original function is "PM", but feels weird to use CAPS for Python functions.